tracing/src/main/java/com/palantir/tracing/Trace.java

```diff
@@ -30,24 +30,35 @@
  */
 public final class Trace {
 
+    private static final Trace NOOP = new Trace(false, "noop");
```

Comment:
Could we put a one-liner of javadoc on this guy to say that we just re-use this single instance for all non-observable traces?